src/Sylius/Component/Core/Model/Order.php

```diff
@@ -468,14 +468,12 @@ public function setShippingState($state)
     }
 
     /**
-     * Gets the last updated shipment of the order
-     *
-     * @return false|ShipmentInterface
+     * {@inheritdoc}
      */
     public function getLastShipment()
```

Comment:
This method should be removed (see [this comment](https://github.com/Sylius/Sylius/pull/6662/files#r87069964)).